app/uk/gov/hmrc/vatregisteredcompaniesstub/models/PayloadSubmissionResponse.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vatregisteredcompaniesstub.models
+
+import java.time.LocalDateTime
+
+import play.api.libs.json.{Json, OFormat}
+
+case class PayloadSubmissionResponse (
+  outcome: String,
```

Comment:
What are the values that this can be? Is it maybe best if this is an enumeration if you know what they all are?

Reply:
Parking for now as discussed